react/modules/module06/README.md

```diff
@@ -1,22 +1,26 @@
 # Week 6
 
-## React. SSR/SSG
+## React. Next.JS/SSR/SSG
 
-Task should be based on the previous task.
+### Disclaimer:
+In this task you going to use Next.js Pages API instead of brand-new APP dir API. The main motivation of it is stability and wide community around this solution. Pages API still supportable by Next.JS team. This decision will help you avoid unnecessary problems and questions regarding server-side components. Anyway, if you are interested in deep dive into APP API you can read about it [here](https://nextjs.org/docs/app/building-your-application/routing) and [here](https://github.com/reactjs/rfcs/blob/main/text/0188-server-components.md).
 
 ### What should be done:
 
 1. Create a separate branch for this task from the previous branch task.
-2. Move your application to the server and enable server side rendering.
-3. Hydrate your application on FE side.
-4. Adopt Redux for SSR.
-5. Add e2e tests for your application via cypress.
+2. Migrate the app to the Next.JS retaining the functionality.
+    - You must enable server-side rendering for all pages.
+3. Switch from react-router to Next.JS router (react-router lib must be removed).
+4. Update tests to make it works with Next.JS.
```

Comment:
Could you change it to "to make it work with" (no s at the end of work)?

Reply:
Fixed